integrations/mixpanel/smart-views/activity-feed/component.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+import SmartViewMixin from 'client/mixins/smart-view-mixin';
+
+export default Ember.Component.extend(SmartViewMixin.default, {
+  currentRecord: null,
+  actions: {
+    selectRecord: function (record) {
+      const currentRecord = this.get('currentRecord');
+
+      if (currentRecord === record) {
+        this.set('currentRecord', null);
+      } else {
+        this.set('currentRecord', record);
+      }
+    }
+  }
+});
```

Comment:
- If this is really what you want to add to the liana, I think it deserves a comment to explain why, because it would be really something new.
- If you didn't want to push such code, you should have detected this yourself.

Anyway this PR in this state is not a 🎁. I'll let you clarify this before going further in the review.